nodes/usb_cam_node.cpp

```diff
@@ -271,5 +280,6 @@ int main(int argc, char **argv)
   ros::init(argc, argv, "usb_cam");
   usb_cam::UsbCamNode a;
   a.spin();
+  std::cout << "finishing" << std::endl;
```

Comment:
looks like they're using ROS_WARN in this file. It's better not two mix the two. Maybe use ROS_INFO?

Reply:
Ah, missed that. That shouldn't even be there, was just using it for debugging.